### PR TITLE
Feature/News-US11: Add edit button for my posted news

### PR DESCRIPTION
**Pull request by wafakd**

### CODE REVIEW

File: webapp/src/main/webapp/news-details/components/ExoNewsDetails.vue [hunk 1]
```diff
@@ -71,17 +81,16 @@
...
 
 export default {
-
   props: {
     news: {
       type: Object,
       required: false,
-      default: function() { return new Object(); }
+      default: function() { return new Object();
+      }
```
Comment (thomasdelhomenie) [inline webapp/src/main/webapp/news-details/components/ExoNewsDetails.vue]: Useless change. To undo.

File: services/src/main/resources/locale/portlet/news/News_fr.properties [hunk 2]
```diff
@@ -114,3 +115,5 @@ news.app.filter.all=Voir toutes
 news.app.filter.pinned=Actualit\u00E9s \u00E9pingl\u00E9es
 
 news.pinned.seeAll=Voir tout
+news.app.in=dans
+news.app.by=par
```
Comment (thomasdelhomenie) [inline services/src/main/resources/locale/portlet/news/News_fr.properties]: These strings are used in News details, not in News app. Let's call them news.activity.in and news.activity.by for the moment to be consistent with others strings.

File: webapp/src/main/webapp/groovy/news/webui/activity/UISharedNewsFullActivity.gtmpl [hunk 3]
```diff
@@ -229,29 +238,35 @@ if (activity && news) {
...
+                         <a class="avatarXSmall" title="$authorFullName" href="$authorProfileURL">
+                      </a>
+                    </div>
+                    <div>
+                      <div class="newsAuthor">
+                        <a href="$authorProfileURL" class="newsInformationValue newsAuthorName"> ${authorFullName} </a>
                       </div>
                       <div>
-                        <span class="newsInformationLabel"> ${labelPublicationDate} :</span>
+                        <span class="newsInformationLabel"> ${labelIn} </span>
```
Comment (thomasdelhomenie) [inline webapp/src/main/webapp/groovy/news/webui/activity/UISharedNewsFullActivity.gtmpl]: Why don't these changes done in UINewsFullActivity.gtmpl as well ?

File: services/src/main/resources/locale/portlet/news/News_fr.properties (hunk 2 above)
Comment (wafakd) [reply services/src/main/resources/locale/portlet/news/News_fr.properties]: ok! done

**Pull request merged**